packages/components/text/src/Text.types.ts

```diff
@@ -11,7 +11,9 @@ export const textName = 'Text';
 export type TextTokens = Omit<FontTokens, 'fontFamily'> &
   IForegroundColorTokens &
   Omit<TextStyle, 'fontSize' | 'fontWeight' | 'color'> & {
-    dynamicTypeRamp?: string; // TODO(#2268): Remove once RN Core properly supports Dynamic Type scaling
+    // TODO(#2268): Remove these once RN Core properly supports Dynamic Type scaling
+    dynamicTypeRamp?: string;
+    maximumFontSize?: number;
```

Comment:
nit" Remove these `iOS specific properties`

I think err on over documenting things to be platform specific

Reply:
Also,I thought the format we followed was `GH #999: Do blah`?

Reply:
Adding documentation should be easy.

Also, I don't see any particular guide for marking TODOs associated with GitHub issues. This should probably be written down somewhere.